client/my-sites/stats/activity-log/index.jsx

```diff
@@ -430,7 +430,7 @@ class ActivityLog extends Component {
 				onPeriodChange={ position === 'bottom' ? this.handlePeriodChangeBottom : changePeriod }
 				period="month"
 				url={ `/stats/activity/${ slug }` }
-				hidePreviousArrow={ 0 === oldestItemTs || monthStartTs <= oldestItemTs }
+				hidePreviousArrow={ Infinity === oldestItemTs || monthStartTs <= oldestItemTs }
```

Comment:
Note that the first condition is unnecessary because the second condition will always be true if the first is true.

```js
Infinity <= Infinity === true
```